utils/constants.py

```diff
@@ -26,4 +26,9 @@
 DEFAULT_IMAGE_END = 'done'
 DEFAULT_IMAGE_END_MESSAGE = {
     'message': DEFAULT_IMAGE_END
+}
+
+NEW_CLIENT_INITIALIZATION = 'client_ids'
+NEW_CLIENT_INITIALIZATION_MESSAGE = {
+    'message' : NEW_CLIENT_INITIALIZATION
 }
```

Comment:
don't call NEW_CLIENT_INITIALIZATION because the content is the topic name, not a message name.

you can create a new section here for topics. you only need NEW_CLIENT_INITIALIZATION_TOPIC = 'client_ids'